packages/apps-v5/README.md

```diff
@@ -20,55 +20,57 @@ USAGE
   $ heroku COMMAND
 ...
 ```
+
 <!-- usagestop -->
 <!-- commands -->
-* [`heroku apps`](#heroku-apps)
-* [`heroku apps:create [APP]`](#heroku-appscreate-app)
-* [`heroku apps:destroy`](#heroku-appsdestroy)
-* [`heroku apps:errors`](#heroku-appserrors)
-* [`heroku apps:favorites`](#heroku-appsfavorites)
-* [`heroku apps:favorites:add`](#heroku-appsfavoritesadd)
-* [`heroku apps:favorites:remove`](#heroku-appsfavoritesremove)
-* [`heroku apps:info`](#heroku-appsinfo)
-* [`heroku apps:open [PATH]`](#heroku-appsopen-path)
-* [`heroku apps:rename NEWNAME`](#heroku-appsrename-newname)
-* [`heroku apps:stacks`](#heroku-appsstacks)
-* [`heroku apps:stacks:set STACK`](#heroku-appsstacksset-stack)
-* [`heroku config:set`](#heroku-configset)
-* [`heroku drains`](#heroku-drains)
-* [`heroku drains:add URL`](#heroku-drainsadd-url)
-* [`heroku drains:remove [URL|TOKEN]`](#heroku-drainsremove-urltoken)
-* [`heroku dyno:kill DYNO`](#heroku-dynokill-dyno)
-* [`heroku dyno:resize`](#heroku-dynoresize)
-* [`heroku dyno:restart [DYNO]`](#heroku-dynorestart-dyno)
-* [`heroku dyno:scale`](#heroku-dynoscale)
-* [`heroku dyno:stop DYNO`](#heroku-dynostop-dyno)
-* [`heroku features`](#heroku-features)
-* [`heroku features:disable FEATURE`](#heroku-featuresdisable-feature)
-* [`heroku features:enable FEATURE`](#heroku-featuresenable-feature)
-* [`heroku features:info FEATURE`](#heroku-featuresinfo-feature)
-* [`heroku keys`](#heroku-keys)
-* [`heroku keys:add [KEY]`](#heroku-keysadd-key)
-* [`heroku keys:clear`](#heroku-keysclear)
-* [`heroku keys:remove KEY`](#heroku-keysremove-key)
-* [`heroku labs`](#heroku-labs)
-* [`heroku labs:enable FEATURE`](#heroku-labsenable-feature)
-* [`heroku labs:info FEATURE`](#heroku-labsinfo-feature)
-* [`heroku maintenance`](#heroku-maintenance)
-* [`heroku maintenance:off`](#heroku-maintenanceoff)
-* [`heroku maintenance:on`](#heroku-maintenanceon)
-* [`heroku notifications`](#heroku-notifications)
-* [`heroku ps [TYPE [TYPE ...]]`](#heroku-ps-type-type-)
-* [`heroku ps:kill DYNO`](#heroku-pskill-dyno)
-* [`heroku ps:resize`](#heroku-psresize)
-* [`heroku ps:restart [DYNO]`](#heroku-psrestart-dyno)
-* [`heroku ps:scale`](#heroku-psscale)
-* [`heroku ps:stop DYNO`](#heroku-psstop-dyno)
-* [`heroku ps:type`](#heroku-pstype)
-* [`heroku releases`](#heroku-releases)
-* [`heroku releases:info [RELEASE]`](#heroku-releasesinfo-release)
-* [`heroku releases:output [RELEASE]`](#heroku-releasesoutput-release)
-* [`heroku releases:rollback [RELEASE]`](#heroku-releasesrollback-release)
+
```

Comment:
Don't need this newline, also lets change the `-` back to `*`, I noticed they all changed for some reason.

Reply:
👍